### ISSUE TITLE
Can we work together on this?

### ISSUE THREAD

**Issue by artfuldev** [cyclejs-community/xstream-marbles#1]
I started a project https://github.com/artfuldev/xstream-marbles which kind of took off from what staltz did with rxmarbles. Do you think we could share ideas or reuse code, and in general, collaborate on this?

**Comment by Widdershin** [comment]
For sure! Either add me as a collaborator to yours, or I can remove this one and you can transfer your repository into the community organisation if you like. Might get even more contributors that way :)

Let's work on what you've got 😄

**Comment by artfuldev** [comment]
Yeah, I was thinking along the same lines. Let me know when I can transfer my repo to this community. We can take it up from there. ☺

**Comment by Widdershin** [comment]
Deleting this repo now, hopefully it will work anytime.
